AccessHandle.md

```diff
@@ -0,0 +1,247 @@
+# Adding AccessHandles to files
+
+## Authors:
+
+* Emanuel Krivoy (fivedots@chromium.org)
+* Richard Stotz (rstz@chromium.org)
+
+## Participate
+
+* [Issue tracker](https://github.com/WICG/file-system-access/issues)
+
+## Table of Contents
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Goals and Use Cases](#goals-and-use-cases)
+- [Non-goals](#non-goals)
+- [Proposed API](#proposed-api)
+  - [New data access surface](#new-data-access-surface)
+  - [Locking semantics](#locking-semantics)
+- [Open Questions](#open-questions)
+  - [Naming](#naming)
+  - [Assurances on non-awaited consistency](#assurances-on-non-awaited-consistency)
+- [Appendix](#appendix)
+  - [AccessHandle IDL](#accesshandle-idl)
+- [Stakeholder Feedback / Opposition](#stakeholder-feedback--opposition)
+- [References & acknowledgements](#references--acknowledgements)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Introduction
+
+This is a proposal to support
+[Storage Foundation API's](https://github.com/WICG/storage-foundation-api-explainer)
+use cases through the Origin Private File System (OPFS), to provide a unified
+interface and a simpler model to developers.
+
+Storage Foundation’s main objective is to support performance-sensitive
+applications through a very fast and generic storage backend. It especially
+focuses on ported WebAssembly programs for which existing storage web APIs
+don’t match the expectations of native code. There are a few design choices
+that primarily contribute to Storage Foundation’s performance:
+
+*   Developers get control of when data is flushed to disk, and reads can be
+    executed consistently without flushing. This allows programs to only schedule
+    time consuming flushes when they are required for persistency, and not as a
+    precondition to operate on recently written data.
+*   Its design ensures that implementations can access data from “the renderer
```

Comment:
I don't see how this is relevant for the Explainer, since the multi-process architecture is Chrome-specific and the quota system might not be part of the spec at all.

Reply:
Removed.